lexer/lexer_test.go

```diff
@@ -95,6 +97,20 @@ Wait+Screen@1m /foobar/`
 		{token.NUMBER, "1"},
 		{token.MINUTES, "m"},
 		{token.REGEX, "foobar"},
+		{token.WAIT, "Wait"},
+		{token.PLUS, "+"},
+		{token.STRING, "Screen"},
+		{token.AT, "@"},
+		{token.NUMBER, "1"},
+		{token.MINUTES, "m"},
+		{token.REGEX, "foo\\/bar"},
```

Comment:
This test expects "foo\\/bar" but the actual string stored should be "foo\/bar" after consuming the escape.